src/test/java/StringTest.java

```diff
@@ -0,0 +1,47 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringTest {
+    @Test
+    @DisplayName("1, 2를 ,로 나누면 각 숫자는 분리된다.")
+    public void split() {
+        String input = "1,2";
+        assertThat(input.split(",")).containsExactly("1", "2");
+    }
+
+    @Test
+    @DisplayName("1을 ,로 나누면 1만 포함하는 배열이 반환된다.")
+    public void splitOneObject() {
+        String input = "1";
+        assertThat(input.split(",")).contains("1");
```

Comment:
```suggestion
        assertThat("1".split(",")).contains("1");
```
로컬 변수 없이 구현하는 것은 어떨까?
너무 극단적일까?